qp_qiime2/tests/test_qiime2.py

```diff
@@ -328,7 +342,7 @@ def test_alpha(self):
 
         # To avoid having to set up all these files, we are gonna test
         # that if phylogenetic and no tree it fails
-        params['i-tree'] = None
+        params['Phylogenetic tree'] = None
```

Comment:
I'm confused, above this is set as `"None"` but now its set as `None`?

Reply:
Changed - given how the code works it was behaving correctly anyways.